app/assets/stylesheets/legacy/chemical-tab.scss

```diff
@@ -130,11 +130,11 @@ table.table-borderless {
     }
       
     .inventory-information-status {
-    width: 22%;
+        width: 28%;
     }
```

Comment:
Looks like many of these classes are not used anymore. Does it make sense to delete them?

Reply:
Hi Ashwini, that might be right, however, it should be handled with regards to the bootstrap fix branch (because the styling of the fields in inventory tab need to be refactored), as this is based on the updates from Bootstrap, not from this PR